### PR TITLE
Explicitly specify linear curves

### PR DESCRIPTION
**Pull request by abarth**
We now required the curve parameter for CurvedAnimation.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/lib/src/material/toggleable.dart [hunk 1]
```diff
@@ -46,7 +46,8 @@ abstract class RenderToggleable extends RenderConstrainedBox implements Semantic
       value: _value ? 1.0 : 0.0
     );
     _position = new CurvedAnimation(
-      parent: _positionController
```
Comment (Hixie) [inline packages/flutter/lib/src/material/toggleable.dart]: this one maybe argues that it's ok to have it imply linear? We do set it later...

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/material/toggleable.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/material/toggleable.dart]: I think it's better to be explicit in this case.  We'll probably want to use `@required` once that works too.

**Pull request merged**